sim/midas/src/main/cc/Makefile

```diff
@@ -39,11 +39,15 @@ $(info clock period: $(CLOCK_PERIOD))
 
 shim := FPGATop
 
-override CXXFLAGS := $(CXXFLAGS) -std=c++11 -Wall -I$(midas_dir)/dramsim2
+testchip_dir=$(TOP_DIR)/generators/testchipip/src/main/resources
```

Comment:
Maybe add a check for `TOP_DIR` being defined, like `DESIGN`/`GEN_DIR`/`DRIVER` above?
```Makefile
ifeq ($(strip $(TOP_DIR)),)
$(error Define TOP_DIR, the path to the root of Chipyard)
endif
```